kratos/python/add_utilities_to_python.cpp

```diff
@@ -331,7 +331,7 @@ void AddUtilitiesToPython(pybind11::module &m)
     InputGetConditionNumber ThisGetConditionNumber = &ConditionNumberUtility::GetConditionNumber;
     DirectGetConditionNumber ThisDirectGetConditionNumber = &ConditionNumberUtility::GetConditionNumber;
 
-    py::class_<ConditionNumberUtility>(m,"ConditionNumberUtility")
+    py::class_<ConditionNumberUtility,ConditionNumberUtility::Pointer>(m,"ConditionNumberUtility")
```

Comment:
why is this needed?
I usually use it without the `Pointer` for simple cases such as this

Reply:
In my application I pass it as an argument, so I need to be a pointer, in general is not necessary

Reply:
ok, understand